client/app/scripts/superdesk-packaging/views/add-to-package.html

```diff
@@ -0,0 +1 @@
+<div sd-add-package-dropdown></div>
```

Comment:
we can revisit this, it would make things more explicit, and technical reasons for that are no longer valid (older ie support)